custom/icds_reports/static/js/directives/info-message/info-message.directive.js

```diff
@@ -29,6 +32,21 @@ function InfoMessageController($location) {
             vm.currentMonth = moment().format("MMMM");
             vm.nextMonth = moment().startOf('month').add(1, 'months').format("MMMM");
             vm.year = moment().format('YYYY');
```

Comment:
I'd recommend setting all variables to `vm.year` to start and then remove all the `else` clauses below

Reply:
changed